### DISCUSSION TITLE
Vuejs

### DISCUSSION THREAD

**Discussion by devmnj** [prisma/prisma#8426] [Q&A] [answered]
Can I use it with Vuejs ?

**Comment by ryands17** [comment, marked as answer]
@manojap 👋 

Prisma cannot be used directly with Vue as it only works on the server side ie. Node. You can use it with a server side framework like [Nuxt](https://nuxtjs.org/)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
